lib/v4/migration-helpers/__tests__/update-api-folder-structure.test.js

```diff
@@ -16,6 +16,13 @@ describe('update api folder structure', () => {
   beforeEach(() => {
     jest.spyOn(console, 'error').mockImplementation(() => {});
     jest.spyOn(console, 'log').mockImplementation(() => {});
+    // Mock the first call to readdir
+    fs.readdir.mockReturnValueOnce([
+      { name: 'test', isDirectory: jest.fn(() => true) },
+      { name: 'test-two', isDirectory: jest.fn(() => true) },
+      { name: 'test.js', isDirectory: jest.fn(() => false) },
+    ]);
+    // Mock the second call to readdir
     fs.readdir.mockReturnValueOnce([
```

Comment:
You could chain the 2 mockReturnValueOnce together like they do in the [docs](https://jestjs.io/docs/mock-function-api#mockfnmockreturnvalueoncevalue)

Reply:
oh yeah good idea 👍